commitlint.config.js

```diff
@@ -1,3 +1,27 @@
+const { maxLineLength } = require('@commitlint/ensure')
+
+const bodyMaxLineLength = 100
+
+const validateBodyMaxLengthIgnoringDeps = parsedCommit => {
+  const { type, scope, body } = parsedCommit
+  const isDepsCommit =
+    type === 'chore' && (scope === 'deps' || scope === 'deps-dev')
+
+  return [
+    isDepsCommit || !body || maxLineLength(body, bodyMaxLineLength),
+    `body's lines must not be longer than ${bodyMaxLineLength}`,
+  ]
+}
```

Comment:
Are you intending to use this to fix the PR failures of the Dependabot updates? 
If that is indeed the case, you should also the line length of the commit header. The longest header at the moment is `conventional-changelog-lint-config-canonical` with up to 89 characters, so is not an real issue quite yet. Good for me either way.

Also, if you don't mind disabling body line length for all commits, you could use my [`vidavidorra/commitlint-config`](https://github.com/vidavidorra/commitlint-config). See the [rules](https://github.com/vidavidorra/commitlint-config/blob/master/docs/rules.md) for details, but in short it disables max line length for everything and allows longer line lengths for deps commits.

The line length isn't specified in [`Conventional Commits v1.0.0`](https://www.conventionalcommits.org/en/v1.0.0/) by the way, so that is something coming from [`commitlint`](https://github.com/conventional-changelog/commitlint/tree/master/%40commitlint/config-conventional).

Reply:
I haven't seen line header issues in the dependabot updates yet, so I'd rather wait for it to appear before adding header line length here as well. I'm using [@commitlint/config-conventional](https://www.npmjs.com/package/@commitlint/config-conventional#header-max-length) which uses `100` for the header length, which has been enough so far :smiley: 

Regarding disabling body line length for all commits, yeah I prefer to only disable it for deps commits, and keep as close to `@commitlint/config-conventional` as possible. I don't have a strong opinion about it actually, I'm also ok letting my terminal/github wrap the lines whenever they see fit. But I like just keep things close to `@commitlint/config-conventional` because I use it here and in other projects.

>The line length isn't specified in Conventional Commits v1.0.0 by the way, so that is something coming from commitlint.

Yeah, linting line lengths is not part of the spec, it's just about personal opinions :wink: